source/chapter2-devicetree-basics.rst

```diff
@@ -26,13 +26,20 @@ devices in a system. Each node has property/value pairs that describe
 the characteristics of the device being represented. Each node has
 exactly one parent except for the root node, which has no parent.
 
-A |spec|-compliant devicetree describes device information in a system
-that cannot necessarily be dynamically detected by a client program. For
-example, the architecture of PCI enables a client to probe and detect
-attached devices, and thus devicetree nodes describing PCI devices
-might not be required. However, a device node is required to describe a
-PCI host bridge device in the system if it cannot be detected by
-probing.
+A device in this context may be an actual hardware device, such as a UART. It
+may be part of a hardware device, such as the random-number generator in a TPM.
+It may also be a purely software concept, such as an bridge providing access to
+an I2C device attached to a remote CPU. There is no requirement that a node in
+a device tree be a physical hardware device.
```

Comment:
No requirement as there are some exceptions, but generally nodes have some correlation to physical h/w devices.

Bindings which are designed 1:1 nodes to $OS drivers are a constant headache. We can't design bindings which cater to just $OS's abstraction (at the current time).

Reply:
OK I tried with some different words